spec/controllers/registrations_controller_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'rails_helper'
+
+RSpec.describe User::RegistrationsController, type: :controller do
+  describe "POST #create" do
+    let!(:valid_attributes_user) do
+      FactoryBot.build(:user).attributes
+                .merge({ password: '123456', confirm_password: '123456' })
```

Comment:
You really need this merge?

Reply:
yes,because without confirmation user will be not create